recipes/turtleFSI/meta.yaml

```
@@ -0,0 +1,56 @@
+{% set name = "turtleFSI" %}
+{% set version = "1.1" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://github.com/KVSLab/{{ name }}/archive/v{{ version }}.tar.gz
+  sha256: 84adf1e4e7a927ee43bd8a18cab75dd08d4d5f21f4b98cac7b920bbe6525bfb9
+
+build:
+  number: 0
+  skip:
```

This must be in a single line to correctly work
```suggestion
  skip: true  # [win or py2k]
```

---

As this doesn't have compiled code, the better practice would be to make this a `noarch: python` package without the `skip` and introduce the Python version constraint in `host` and `run`. If this is failing on Windows with missing dependencies in staged-recipes, this is ok. On the feedstock, we will build on Linux only.

---

Thank you for your comment, I did not know that it was ok to have missing dependencies for a platform or only support py3 when using `noarch:python`. This is now changed.

---

@xhochy When setting `noarch:python` the build on windows tests fail, however, this is expected behavior. Is there something I should do to make these tests pass?

---

If it fails due to missing dependencies, then this is ok here.